config-linux.md

```diff
@@ -171,14 +171,15 @@ Also known as cgroups, they are used to restrict resource usage for a container
 cgroups provide controls (through controllers) to restrict cpu, memory, IO, pids and network for the container.
 For more information, see the [kernel cgroups documentation][cgroup-v1].
 
-The path to the cgroups can be specified in the Spec via `cgroupsPath`.
-`cgroupsPath` can be used to either control the cgroup hierarchy for containers or to run a new process in an existing container.
-If `cgroupsPath` is:
-* ... an absolute path (starting with `/`), the runtime MUST take the path to be relative to the cgroup mount point.
-* ... a relative path (not starting with `/`), the runtime MAY interpret the path relative to a runtime-determined location in the cgroup hierarchy.
-* ... not specified, the runtime MAY define the default cgroup path.
+#### <a name="configLinuxCgroupsPath" />Cgroups Path
+
+**`cgroupsPath`** (string, OPTIONAL) path to the cgroups. It can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.The value of `cgroupsPath` is either an absolute path or a relative path.
```

Comment:
I'd recommend [one sentence per line][1], e..g see #811 where you split multi-sentence lines in a few places.  And if you don't split the sentences onto their own lines, you'll want a space before “The value”.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/style.md#one-sentence-per-line